src/main/java/no/nav/pto/veilarbportefolje/oppfolging/OppfolgingStartetService.java

```diff
@@ -32,4 +36,8 @@ public boolean shouldRewind() {
     public void setRewind(boolean rewind) {
 
     }
+
+    private boolean erPostgresPa() {
```

Comment:
Maybe you can move this function erPostresPa to one place and make it maybe static, instead of having it in multiple places.

Reply:
was the last change better?